### DISCUSSION TITLE
请教几个关于扇区故障的问题

### DISCUSSION THREAD

**Discussion by shadowlinyf** [filecoin-project/community-china#202] [新手上路]
1.如果扇区中有用户数据，然后扇区因为硬盘坏了导致数据丢失，如果没法恢复硬盘中的数据的话，是不是只能叫扇区故障费和扇区终止费了？
2.如果扇区中没有用户数据，只是承诺扇区的话。在没有办法恢复硬盘数据的情况下，是否有可能通过重建一个一模一样的扇区来恢复？
谢谢

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
